### PR TITLE
Feature/FortiSwitch Module Update

### PR DESCRIPTION
**Pull request by abprasa**
# Description
The existing code has a generic FortiSwitch module, this new code updates the existing FortiSwitch Module, adds more modules based on the currently available releases, and maps them to the relevant versions and the features they support.

# Impacts
Making it more suitable for users to choose modules based on the version of the switch. These new modules will only have the supported features enabled

# NEW Package(s) required
No additional packages are needed. 

# Feature
The idea of updating the existing list of FortiSwitch Modules defined based on the model number to respective versions is to make it more convenient for users to choose modules based on the version of the switch.

# Delete branch after the merge
YES 

# Checklist
- [ ] Document the feature In the Network Device Config Guide
- [ ] Migration Script
- [ ] Test the new modules on UI

# NEWS file entries
## New Features
* Updating existing switch model for the next release

### CODE REVIEW

**Comment by satkunas** [comment]
@fdurand @lzammit review docs

**Comment by extrafu** [comment]
@abprasa Migration script needed to rename the switch module being used in switches.conf (and potentially other places) during an upgrade.

**Comment by abprasa** [comment]
> @abprasa Migration script needed to rename the switch module being used in switches.conf (and potentially other places) during an upgrade.

Sure @extrafu thanks, I was able to create and update the migration script for Fortiswitch and I will get it reviewed with @fdurand

File: docs/network/networkdevice/fortinet_switches.asciidoc [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+       end
+       set security-groups <security-group-name>
+     end
+
+===== Viewing the 802.1x Details
+
+Using the CLI:
+Use the following command to show diagnostics on one or all ports:
+
+ diagnose switch 802-1x status [<port>]
```
Comment (JeGoi) [inline docs/network/networkdevice/fortinet_switches.asciidoc:149]:
```suggestion
[source,bash]
----
 diagnose switch 802-1x status [<port>]
----
```
To be able to copy/paste

File: docs/network/networkdevice/fortinet_switches.asciidoc [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+	Sessions info:
+	f0:4d:a2:be:a3:31 Type=802.1x,TTLS,state=AUTHENTICATED,etime=0,eap_cnt=9 params:reAuth=60
+	user="local-RADIUS",security_grp="radiusgrp",fortinet_grp="Radius_Admins"
+
+===== Clearing Authorized Sessions
+
+To use the CLI to clear the authorized sessions associated with an interface:
+
+ execute 802-1x clear interface {internal | <port_name>}
```
Comment (JeGoi) [inline docs/network/networkdevice/fortinet_switches.asciidoc:187]:
```suggestion
[source,bash]
 ----
 execute 802-1x clear interface {internal | <port_name>}
----
```
Same, improve the user action

File: docs/network/networkdevice/fortinet_switches.asciidoc [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+For example:
+
+ execute 802-1x clear interface port3
+
+To use the CLI to clear the authorized session associated with a MAC address:
+
+ execute 802-1x clear mac <MAC_address>
```
Comment (JeGoi) [inline docs/network/networkdevice/fortinet_switches.asciidoc:195]:
```suggestion
[source,bash]
---
 execute 802-1x clear mac <MAC_address>
---
```
Same help user to copy/paste

**Review by JeGoi — COMMENTED** [review]
Just check if comment on documentation worth it.
Then agree to merge

**Comment by JeGoi** [comment]
In fact, the file name from addons/upgrade/to-13.0-convert-fortiswitch-types.pl to addons/upgrade/to-13.1-convert-fortiswitch-types.pl before merging

**Comment by abprasa** [comment]
> In fact, the file name from addons/upgrade/to-13.0-convert-fortiswitch-types.pl to addons/upgrade/to-13.1-convert-fortiswitch-types.pl before merging

Sure @JeGoi will do.

**Comment by abprasa** [comment]
> In fact, the file name from addons/upgrade/to-13.0-convert-fortiswitch-types.pl to addons/upgrade/to-13.1-convert-fortiswitch-types.pl before merging

done

**Review by JeGoi — APPROVED** [review]

**Pull request merged**